storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2676,6 +2677,14 @@ private void assertTopoActive(String topoName, boolean expectActive) throws NotA
         return tryReadTopoConf(topoId, topoCache);
     }
 
+    private StormTopology tryReadTopologyFromName(final String topoName) throws NotAliveException,
+            AuthorizationException, IOException {
+        IStormClusterState state = stormClusterState;
+        String topoId = state.getTopoId(topoName)
```

Comment:
can be replaced by https://github.com/apache/storm/blob/master/storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java#L1789-L1792

Reply:
Replaced.